.github/workflows/release.yml

```
@@ -2,77 +1,29 @@
+name: Github release 
 
 on:
-  workflow_dispatch:
-    inputs:
-      name:
-        description: 'Version name'
-        required: true
-        type: string
+    workflow_call:
+      inputs:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)